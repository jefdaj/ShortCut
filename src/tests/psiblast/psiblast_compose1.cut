# psiblast functions that use compose1 function
# (this is an implementation detail, not something important to know as a user)

# setup
query   = load_faa "data/7942gene.faa"
pcc6803 = load_faa "data/6803genes.faa"
pcc7942 = load_faa "data/7942genes.faa"
db6803  = makeblastdb_prot pcc6803
db7942  = makeblastdb_prot pcc7942
pssms   = psiblast_train_db_each 1e-2 query [db7942, db6803]

# compose1-based fns to test
table1 = psiblast_pssms_db  1e-2 pssms  db7942
table2 = psiblast_pssms_all 1e-2 pssms pcc7942

# don't do this with real data unless comparable
result = concat_bht [table1, table2]
