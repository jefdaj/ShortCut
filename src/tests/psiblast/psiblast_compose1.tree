.
|-- cache
|   |-- lines
|   |   |-- 14c596330b.txt
|   |   |-- 33ec325d48.txt
|   |   |-- 3ec2030ccb.txt
|   |   |-- 582b6034f3.txt
|   |   |-- 74a09cfaee.txt
|   |   |-- 86c900eddd.txt
|   |   |-- 8abd22b6c2.txt
|   |   |-- 902cb58552.txt
|   |   |-- 9076e70b45.txt
|   |   |-- 908f92d67a.txt
|   |   |-- a1512fc008.txt
|   |   |-- a27012ecc8.txt
|   |   |-- adfdb6bd8c.txt
|   |   |-- bdc783547b.txt
|   |   |-- d3e3db2352.txt
|   |   `-- e00fb07ddb.txt
|   |-- load
|   |   |-- 82645ed6f6.faa -> $WORKDIR/data/7942gene.faa
|   |   |-- 8aa4a9d2fc.faa -> $WORKDIR/data/6803genes.faa
|   |   `-- a810086108.faa -> $WORKDIR/data/7942genes.faa
|   |-- makeblastdb
|   |   |-- a1512fc008
|   |   |   |-- a1512fc008.pdb.phr
|   |   |   |-- a1512fc008.pdb.pin
|   |   |   `-- a1512fc008.pdb.psq
|   |   `-- e00fb07ddb
|   |       |-- e00fb07ddb.pdb.list.phr
|   |       |-- e00fb07ddb.pdb.list.pin
|   |       |-- e00fb07ddb.pdb.list.psq
|   |       |-- e00fb07ddb.pdb.phr
|   |       |-- e00fb07ddb.pdb.pin
|   |       `-- e00fb07ddb.pdb.psq
|   `-- map
|       |-- 10ff2e7e17.pssm -> ../../cache/lines/86c900eddd.txt
|       |-- 50b1efde93.bht
|       |-- 56fbede985.bht
|       |-- 6d0199cc7a.bht
|       |-- 7ca6987be7.bht
|       `-- d84f579869.pssm -> ../../cache/lines/9076e70b45.txt
|-- exprs
|   |-- concat_bht
|   |   |-- 0c8e7ecb05.bht
|   |   |-- 1f2a079cfd.bht
|   |   `-- f55412aae2.bht
|   |-- list
|   |   |-- 94f9daed94.faa.list -> ../../cache/lines/a1512fc008.txt
|   |   |-- 970ceb8761.pdb.list -> ../../cache/lines/d3e3db2352.txt
|   |   |-- b3693202c4.bht.list -> ../../cache/lines/14c596330b.txt
|   |   `-- fa4677789c.faa.list -> ../../cache/lines/e00fb07ddb.txt
|   |-- load_faa
|   |   |-- 33c746245c.faa -> ../../cache/load/82645ed6f6.faa
|   |   |-- 387f499ddc.faa -> ../../cache/load/a810086108.faa
|   |   `-- dee24d6c3c.faa -> ../../cache/load/8aa4a9d2fc.faa
|   |-- makeblastdb_prot
|   |   |-- 27a934c58e.pdb -> ../../cache/lines/908f92d67a.txt
|   |   |-- ced061f0b2.pdb -> ../../cache/lines/902cb58552.txt
|   |   `-- ced061f0b2.pdb.list -> ../../cache/lines/74a09cfaee.txt
|   |-- num
|   |   `-- c23938f494.num -> ../../cache/lines/33ec325d48.txt
|   |-- psiblast_each_pssm
|   |   `-- f51c91829c_1ebf035366_8a72f50db3.bht.list -> ../../cache/lines/3ec2030ccb.txt
|   |-- psiblast_each_pssm_db
|   |   `-- f51c91829c_1ebf035366_aefb289407.bht.list -> ../../cache/lines/582b6034f3.txt
|   |-- psiblast_train_db_each
|   |   `-- f51c91829c_4d0e43d616_3109b90cab.pssm.list -> ../../cache/lines/a27012ecc8.txt
|   `-- str
|       |-- 1d8a61c770.str -> ../../cache/lines/bdc783547b.txt
|       |-- b9eaf59b0c.str -> ../../cache/lines/8abd22b6c2.txt
|       `-- ec3b181d8a.str -> ../../cache/lines/adfdb6bd8c.txt
`-- vars
    |-- db6803.pdb -> ../exprs/makeblastdb_prot/27a934c58e.pdb
    |-- db7942.pdb -> ../exprs/makeblastdb_prot/ced061f0b2.pdb
    |-- pcc6803.faa -> ../exprs/load_faa/dee24d6c3c.faa
    |-- pcc7942.faa -> ../exprs/load_faa/387f499ddc.faa
    |-- pssms.pssm.list -> ../exprs/psiblast_train_db_each/f51c91829c_4d0e43d616_3109b90cab.pssm.list
    |-- query.faa -> ../exprs/load_faa/33c746245c.faa
    |-- result -> ../exprs/concat_bht/0c8e7ecb05.bht
    |-- table1.bht -> ../exprs/concat_bht/1f2a079cfd.bht
    `-- table2.bht -> ../exprs/concat_bht/f55412aae2.bht

18 directories, 63 files
