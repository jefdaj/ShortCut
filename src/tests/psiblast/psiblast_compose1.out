Synpcc7942_0002	Synpcc7942_0002	100.000	290	0	0	193	482	1	290	0.0	586
Synpcc7942_0002	Synpcc7942_2044	30.556	72	47	1	283	351	91	162	0.005	35.4
Synpcc7942_0002	Synpcc7942_0002	100.000	290	0	0	193	482	1	290	0.0	586
Synpcc7942_0002	Synpcc7942_2044	30.556	72	47	1	283	351	91	162	0.005	35.4
Synpcc7942_0002	Synpcc7942_0002	100.000	290	0	0	193	482	1	290	0.0	586
...

